app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabHomeFeature.kt

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.recentsyncedtabs
+
+import android.content.Context
+import androidx.lifecycle.LifecycleOwner
+import androidx.navigation.NavController
+import mozilla.components.browser.storage.sync.SyncedDeviceTabs
+import mozilla.components.feature.syncedtabs.SyncedTabsFeature
+import mozilla.components.feature.syncedtabs.storage.SyncedTabsStorage
+import mozilla.components.feature.syncedtabs.view.SyncedTabsView
+import mozilla.components.feature.tabs.TabsUseCases
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.appstate.AppAction
+import org.mozilla.fenix.home.HomeFragmentDirections
+import org.mozilla.fenix.home.recentsyncedtabs.controller.RecentSyncedTabController
+import org.mozilla.fenix.tabstray.Page
+
+/**
+ * HomeFragment delegate to handle layout updates and dispatch actions related to the recent
+ * synced tab.
+ *
+ * @param store Store to dispatch actions to when synced tabs are updated or errors encountered.
+ * @param addNewTabUseCase Use case to open the synced tab when clicked.
+ * @param navController Nav controller to navigate to synced tabs tray.
+ * @param accountManager Account manager used to retrieve synced tab state.
```

Comment:
This is a super pedantic stance of mine, but I'd argue from the context of a client of the class, which is calling the constructor and arguably who the kdoc is for, has no knowledge of whether the constructor parameter is a property of the class or not since it's private. 

I'll switch it here, just wanted to drop my two cents for fun